h/static/styles/partials-v2/_search.scss

```diff
@@ -115,9 +115,10 @@
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
+  min-width: 0;  // Make sure the .search-bucket-stats element's content wraps
```

Comment:
Note that `word-wrap` is a widely supported but non-standard alias for `overflow-wrap`. See https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap